pkg/virt-launcher/virtwrap/access-credentials/access_credentials_test.go

```diff
@@ -237,21 +237,20 @@ var _ = Describe("AccessCredentials", func() {
 		domName := util.VMINamespaceKeyFunc(vmi)
 
 		manager.watcher, err = fsnotify.NewWatcher()
-		Expect(err).To(BeNil())
+		Expect(err).ToNot(HaveOccurred())
 
 		secretDirs := getSecretDirs(vmi)
-		Expect(len(secretDirs)).To(Equal(1))
+		Expect(secretDirs).To(HaveLen(1))
 		Expect(secretDirs[0]).To(Equal(fmt.Sprintf("%s/%s-access-cred", tmpDir, secretID)))
 
 		for _, dir := range secretDirs {
-			os.Mkdir(dir, 0755)
-			err = manager.watcher.Add(dir)
-			Expect(err).To(BeNil())
+			Expect(os.Mkdir(dir, 0755)).To(Succeed())
+			Expect(manager.watcher.Add(dir)).To(Succeed())
 		}
 
 		// Write the file
 		err = ioutil.WriteFile(filepath.Join(secretDirs[0], user), []byte(password), 0644)
-		Expect(err).To(BeNil())
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
This fit a direct approach with `.To(Succeed())`

Reply:
Done